inflammation/models.py

```diff
@@ -54,3 +54,13 @@ def daily_min(data):
     """Calculate the daily min of a 2d inflammation data array."""
     return np.min(data, axis=0)
 
+
+def s_dev(data):
```

Comment:
Not the clearest function name, could rename to `standard_deviation` instead

Reply:
How true that is! The function will be renamed to standard_deviation